Cargo.toml

```diff
@@ -2,7 +2,7 @@
 
 members = [
     "common",
```

Comment:
Could you rename to era-compiler-common?

Reply:
Done. Please, check.